validator/client/src/main/java/tech/pegasys/teku/validator/client/AttestationDutyLoader.java

```diff
@@ -147,10 +161,18 @@ private SafeFuture<Void> scheduleAggregation(
       final Validator validator,
       final UInt64 slot,
       final int aggregatorModulo,
-      final SafeFuture<Optional<AttestationData>> unsignedAttestationFuture) {
+      final SafeFuture<Optional<AttestationData>> unsignedAttestationFuture,
+      final Optional<DvtAttestationAggregations> dvtAttestationAggregation) {
     return forkProvider
         .getForkInfo(slot)
         .thenCompose(forkInfo -> validator.getSigner().signAggregationSlot(slot, forkInfo))
+        .thenCompose(
+            blsSignature ->
+                dvtAttestationAggregation
+                    .map(
+                        dvt ->
+                            dvt.getCombinedSelectionProofFuture(validatorIndex, slot, blsSignature))
```

Comment:
lets maybe add a log.debug in here showing us the index and slot in case we need it when remote debugging...
unless that dvt object maybe has one?

Reply:
nvm it does.